docs/drivers.md

```diff
@@ -35,7 +44,8 @@ Now install the driver:
 ```shell
 curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2 \
   && sudo install docker-machine-driver-kvm2 /usr/local/bin/
-```
+
```

Comment:
What happened here ? Seems like the "shell" end was lost ?

Reply:
yeah it was. i cleaned it up .